types/simple-element-resize-detector/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for simple-element-resize-detector 1.3
+// Project: https://github.com/developit/simple-element-resize-detector#readme
+// Definitions by: Nathan Bierema <https://github.com/Methuselah96>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function resizeDetector(
+    element: HTMLElement,
+    handler: () => void
+): HTMLIFrameElement;
+export = resizeDetector;
```

Comment:
```suggestion
export as namespace simpleElementResizeDetector;

declare function simpleElementResizeDetector(
    element: HTMLElement,
    handler: () => void
): HTMLIFrameElement;
export = simpleElementResizeDetector;
```

this seems to be a proper UMD with exported globla reference for resize detector, so for this reason I'd recommend accepting the proposed name changes.
Doing that would allow to use on the website script directly:
```ts
const element = document.body;

simpleElementResizeDetector(element, () => {
	console.log('new size: ', {
		width: element.clientWidth,
		height: element.clientHeight
	});
});
```